docs/integrations/google/workspace/collect-logs.md

```diff
@@ -198,3 +200,496 @@ _sourceCategory=google*
 | count by email
 | sort by _count desc, email asc
 ```
+
+
+## Configure Collection for Google Workspace Alert Center
```

Comment:
This section is for the legacy AlertCenter collection so we should remove it and replace with the new C2C Google Workspace AlertCenter one.

Reply:
Fixed in file in PR.